crates/forge/tests/data/max_steps/tests/with_attr.cairo

```diff
@@ -0,0 +1,15 @@
+use max_steps::fib;
+
+#[test]
+#[max_steps(120)]
+fn hard_test() {
+    fib(0, 1, 30344);
+    assert(2 == 2, 'simple check');
```

Comment:
That's kind of a bummer that we have to make it panicable by adding this assertion, since `max_steps` is kind of an assertion

Reply:
Can we perhaps make a task to make it work like an assertion (so we don't require the panicable return type of test function)?

Reply:
Idk if it is worth doing and if it is doable, I believe it has to be panicable to get the `RunResult` in `run_test_case`, but maybe I'm wrong
cc @MaksymilianDemitraszek

Reply:
I believe it could be relaxed 🤔 not sure if this is worth the effort atm though. Could we create an issue?